pkg/detectors/aiven/aiven.go

```diff
@@ -21,7 +22,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"aiven"}) + `([a-zA-Z0-9/+=]{372})`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"aiven"}) + `\b([a-zA-Z0-9/+=]{372})\b`)
```

Comment:
The previous regex fails detecting our test secret as does the new one. This one needs more work still

Reply:
Just update the number of chars to 372 to 380, Also boundary won't work for this as it contains special chars within the match/key.

Reply:
> Just update the number of chars from 372 to 380

I'm not following how this would solve the problem.

Reply:
Generated a couple tokens just now and they all have `377` characters

Reply:
alas, the problem still persists https://regex101.com/r/nMLU8f/1. Possible solution: https://github.com/trufflesecurity/trufflehog/pull/2407#discussion_r1484822724

Reply:
This is dumb, but try adding a `?` before the `\b`. That works in your linked example at least.

Reply:
Super weird last month I had tried multiple times it was generating  keys of `380` length each time. Will try again.

Reply:
Just confirmed it, Its still generating keys of `380` length for me. 
Just to be sure you are creating tokens on https://console.aiven.io/profile/tokens page right?

https://regex101.com/r/MSXILB/1

Reply:
>  Just to be sure you are creating tokens on https://console.aiven.io/profile/tokens page right?

Yep. Try creating an account with another email and generating some tokens

Reply:
Super Weird, Tried with new account and now its generating token of `372` length

Reply:
It's probably encoding account-specific data, which may explain the variance.

Reply:
> This is dumb, but try adding a ? before the \b. That works in your linked example at least.

Lazy loading does the trick. Good eye @rgmz